src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/Dependency.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+option java_multiple_files = true;
+option java_package = "com.microsoft.azure.functions.rpc.messages";
+option java_outer_classname = "DependencyProto";
+option csharp_namespace = "Microsoft.Azure.WebJobs.Script.Grpc.Messages";
```

Comment:
Please address feedback in PR: https://github.com/Azure/azure-functions-language-worker-protobuf/pull/24